src/PolarSSL++/BlockingSslClientSocket.cpp

```diff
@@ -305,7 +306,7 @@ void cBlockingSslClientSocket::OnConnected(void)
 
 void cBlockingSslClientSocket::OnConnectError(const AString & a_ErrorMsg)
 {
-	LOG("Cannot connect to %s: %s", m_ServerName.c_str(), a_ErrorMsg.c_str());
+	LOG("Cannot connect to %s: \"%s\"", m_ServerName.c_str(), a_ErrorMsg.c_str());
```

Comment:
They are, indeed, needed. If the error message is empty for any reason, it will not produce a misleading output.